README.md

```diff
@@ -108,18 +108,18 @@ Use Dataflow to calculate per-user scores and write them to BigQuery.
 What is the total score of the user 'user0_AmberDingo'?
 
 1.  Modify
-    `src/main/java8/com/google/cloud/dataflow/tutorials/game/Exercise1.java`
+    `src/main/java8/org/apache/beam/examples/complete/game/Exercise1.java`
 1.  Run the pipeline (using Direct runner)
 
     ```shell
     $ mvn compile exec:java \
-         -Dexec.mainClass=Exercise1 \
+         -Dexec.mainClass=org.apache.beam.examples.complete.game.Exercise1 \
          -Dexec.args="--project=YOUR-PROJECT \
                       --tempLocation=gs://YOUR-STAGING-BUCKET \
                       --runner=DirectRunner \
                       --outputDataset=YOUR-BIGQUERY-DATASET \
                       --outputTableName=user_scores \
-                      --input=gs://dataflow-sme-tutorial/gaming_data0.csv"
+                      --input=gs://dataflow-sme-tutorial/gaming_data1.csv"
```

Comment:
Doesn't gaming_data1.csv contain much more data than gaming_data0.csv? I'd advise against use for the DirectRunner use-case in that case.